src/main/java/org/folio/services/invoice/InvoiceCancelService.java

```diff
@@ -121,7 +121,7 @@ private CompletableFuture<Void> cancelTransactions(String invoiceId, List<Transa
         logger.error("Failed to cancel transactions for invoice with id {}", invoiceId, t);
         List<Parameter> parameters = Collections.singletonList(
           new Parameter().withKey("invoiceId").withValue(invoiceId));
-        throw new HttpException(500, CANCEL_TRANSACTIONS_ERROR.toError().withParameters(parameters));
+        throw new HttpException(400, CANCEL_TRANSACTIONS_ERROR.toError().withParameters(parameters));
```

Comment:
@SinghAdes I think I mixed up the links in original story description, in this palce we should use 500 status code since it's unexpected situation.
Here is place where we should not use 500 status code, better to use 400
https://github.com/folio-org/mod-invoice/blob/master/src/main/java/org/folio/rest/impl/InvoiceLineHelper.java#L404 

Also please look into another places of source code, maybe we will found somethinf else

Reply:
fixed